components/molecules/ContributorListTableRow/contributor-list-table-row.tsx

```diff
@@ -47,12 +47,9 @@ function getLastContributedRepo(pullRequests: DbRepoPR[]) {
 function getTopContributorLanguages(contributor: DbUser) {
   // some contributors will have empty language objects so we will pull their popular language from the interests field instead of defaulting to nothing
   const entries = Object.entries<string>(contributor.languages);
-  if (entries.length === 0) {
-    return contributor.interests.split(",").slice(0, 2);
-  }
 
   return entries.length === 0
-    ? contributor.interests.split(",").slice(0, 2)
+    ? contributor.interests.split(",")
     : entries
         .sort(([, a], [, b]) => (a < b ? -1 : 1))
         .slice(0, 2)
```

Comment:
This will still only slice and sort entries, we need the parentheses.

```diff
-  return entries.length === 0
-    ? contributor.interests.split(",").slice(0, 2)
-    ? contributor.interests.split(",")
-    : entries
+  return (entries.length === 0 ? contributor.interests.split(",") : entries)
    .sort(([, a], [, b]) => (a < b ? -1 : 1))
    .slice(0, 2)
    .map(([language]) => language);
```